.vscode/tasks.json

```diff
@@ -1,20 +1,31 @@
 {
-  "version": "0.1.0",
+  "version": "2.0.0",
```

Comment:
Please don't bump the version

Reply:
Sorry, I'm not sure why it got bumped b/c I didn't touch it...

Reply:
Oh, nm. This is actually okay. I somehow thoughts this was the package.json.